plugins/1x.js

```diff
@@ -24,6 +24,31 @@ hoverZoomPlugins.push({
             '-thumb.',
             '-fullsize.'
         );
+		hoverZoom.urlReplace(res,
```

Comment:
Please use spaces instead of tabs in the source code. Take a look how this diff looks in github (https://github.com/extesy/hoverzoom/pull/179/files) and you'll easily see the difference.